src/App.scss

```diff
@@ -3,18 +3,81 @@ html {
 }
 
 .App {
+  width: 1200px;
+  margin: 0 auto;
+
   &__header {
+    position: relative;
     display: flex;
     align-items: center;
     gap: 20px;
+    justify-content: center;
+
+    color: white;
+    background-color: rgb(245, 14, 14);
+    border-radius: 20px;
+  }
+
+  &__clear {
+    position: absolute;
+    right: 45px;
+  }
+
+  &__button {
+    width: 100px;
+    padding: 10px 15px;
+
+    font-weight: bold;
+    font-size: 16px;
+
+    background-color: #fff000;
+    border: none;
+    border-radius: 12px;
+
+    cursor: pointer;
+
+    transition: 200ms;
+
+    &:hover {
+      outline: 0;
+      background: #f4e603;
+      box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.2);
+    }
+  }
+
+  &__list {
+    padding: 0;
+    margin: 2px 25px;
+
+    background-color: antiquewhite;
+    border: 5px double black;
   }
 }
 
 .Good {
   display: flex;
   gap: 10px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+
+  font-size: 20px;
+
+  transition: 0.3s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.